edflow/data/dataset_mixin.py

```diff
@@ -92,6 +98,12 @@ def __init__(self):
 
     **Labels in the example ``dict``**
 
+    Oftentimes it is good to store and load some values as lables as it can
+    increase performance and decrease storage size, e.g. when storing scalar
+    D = ConcatenatedDataset(A, A, A)  # Multiplying two datasets
```

Comment:
did something go wrong here? when storing scalar D = ConcatenatedDataset ? Also the text is duplicated below in line 107?

Reply:
Yeah, that was weird. It's fixed now.